.travis.yml

```diff
@@ -15,21 +15,35 @@ env:
 
 matrix:
   fast_finish: true
+  #power on support added
+  include:
+    - os: linux
+      env: TARGET=ppc64le
+      go: 1.13.x
+      arch: ppc64le
+    - os: linux
+      env: TARGET=ppc64le
+      go: 1.14.x
+      arch: ppc64le
+
 
 install:
  - go get golang.org/x/tools/cmd/cover
  - go get github.com/modocache/gover
  - go get github.com/mattn/goveralls
  - go get -t ./...
 
+
 script:
  - >
-    if [ "${TARGET}" == "amd64" ]; then
+   if [ $TARGET == :amd64: ] || [ $TARGET == "ppc64le" ]; then
```

Comment:
This should be something like
`if [[ "${TARGET}" == $(go env GOARCH) ]]; then`

Reply:
@squeed taken care now as you expected